freshrss/umbrel-app.yml

```diff
@@ -0,0 +1,73 @@
+manifestVersion: 1
+id: freshrss
+category: networking
+name: freshrss
+version: "1.21.0"
+tagline: A free, self-hostable aggregator for rss feeds.
+description: >-
+  FreshRSS is a self-hosted RSS and Atom feed aggregator.
+   It is lightweight, easy to work with, powerful, and customizable.
```

Comment:
Let's keep this description short and sweet. We can just keep it as the official
```
FreshRSS is a self-hosted RSS and Atom feed aggregator. It is lightweight, easy to work with, powerful, and customizable.
```

When there's too much in the description it can look a little sloppy in the UI:

<img width="1318" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/eeba774e-2d6f-4d98-a44e-45012699b293">

If we want to add some of the features we could add simple bullet points

```
FreshRSS is a self-hosted RSS and Atom feed aggregator. It is lightweight, easy to work with, powerful, and customizable.

- Follow websites, podcasts and video channels in a single place.

- Search and save queries for quick access.

etc...
```

Reply:
I have implemented this change

Reply:
Thanks. The `etc...` was there to indicate that we can keep adding bullet points. Sorry for the confusion.

I actually ended up changing up the description so that it mirrors this nice one they have in their Docs: https://freshrss.github.io/FreshRSS/en/